config/etcd-io/org.yaml

```diff
@@ -122,15 +132,6 @@ teams:
     privacy: closed
     repos:
       website: admin
-  maintainers-auger:
-    description: Granted write access to auger
-    members:
-    - jmhbnz
-    - siyuanfoundation
-    - wenjiaswe
-    privacy: closed
-    repos:
-      auger: admin
   members:
     description: ""
     members:
```

Comment:
Please also add @ivanvc to the members team so they get the appropriate triage permissions for etcd, thanks.

Reply:
I’d suggest this as as second step once @ivanvc accepts the request and joins the org.

@Priyankasaggu11929, @MadhavJivrajani, please advise

Reply:
@jasonbraganza, yes, adding folks to GH teams is done after org membership is in place.

For now, I think we can add @ivanvc to the requested GH team in a separate commit in this PR and Peribolos should be able to reconcile that.

@jmhbnz, could you please mention the name of the GH teams that @ivanvc needs access to? Thanks!

Reply:
Sure thing!  Team is `members`.  Refer diff line 134-135

Reply:
@Priyankasaggu11929, done. Please take a look.

Reply:
I'm not seeing `ivanvc` in the members team @jasonbraganza?

Reply:
Well I’ll be … sorry folks! will go have a look see later in the day. apologies!

Reply:
think I licked it. @Priyankasaggu11929, now will you ptal?